contrib/pkg/createcluster/aws.go

```diff
@@ -56,14 +60,12 @@ func (p *awsCloudProvider) addPlatformDetails(o *Options, cd *hivev1.ClusterDepl
 		},
 	}
 
-	cd.Spec.Compute[0].Platform = hivev1.MachinePoolPlatform{
-		AWS: &hivev1aws.MachinePoolPlatform{
-			InstanceType: "m4.xlarge",
-			EC2RootVolume: hivev1aws.EC2RootVolume{
-				IOPS: 100,
-				Size: 22,
-				Type: "gp2",
-			},
+	machinePool.Spec.Platform.AWS = &hivev1aws.MachinePoolPlatform{
+		InstanceType: "m4.xlarge",
```

Comment:
Is there anyway not to hard code the instanceType and use the default value from installer? I throught we added the code not to hard code this. Also if we have to hard code then we should use hard code these in one place and reuse the variables here. In that case we can change the default values easily if we want to.

Reply:
Unfortunately, we do not have a good story around using the installer defaults with MachinePools. For now, we've made the decision to require that the user supply values for the instance type. You can look in slack history to read the conversation that was had regarding this, with a focus on gcp.

I will add a constant to the file.